### DISCUSSION TITLE
Getting embeddings from only one speaker in a recording with two

### DISCUSSION THREAD

**Discussion by Rahul-Brito** [pyannote/pyannote-audio#924] [Q&A]
Hello! Sorry if this should be a feature or bug request, but it just felt like a question I was stuck on!

- I had previously been using this [example](https://github.com/pyannote/pyannote-audio/tree/master/tutorials/pretrained/model). I thought I was using an older version of pyannote-audio though it looks like my version is 2.0 in that virtual environment, so maybe this is an older version of the API? Sorry if I am confused
- I started playing around with some of the new pretrained [models](https://huggingface.co/models?other=pyannote-audio-model) and [pipelines](https://huggingface.co/models?other=pyannote-audio-pipeline). The new APIs look a lot slicker! Specifically, I used the new [embedding pipeline](https://huggingface.co/pyannote/embedding). However, I was only able to get one embedding vector for one .wav file, when in the previous example it would calculate embeddings on each 500ms segment of recording where there was speech, which seemed more optimal. Is this just a user-error or a change in the API?
- Part of the reason I ask is because I could easily crop my recording too. In this case, when I use the [speaker diarization ](https://huggingface.co/pyannote/speaker-diarization) code, I can see that there are two speakers in a recording, one of whom I want to cut out (one is the participant, one is someone giving directions who accidentally recorded themselves). I can do so easily with the label_timeline() function, but then I couldn't figure out how to just get the embeddings from this speaker anymore (e.g. with embeddings.crop)

**Comment by hbredin** [comment]
See section [extracting embeddings using a sliding window](https://huggingface.co/pyannote/embedding#extract-embeddings-using-a-sliding-window). `embeddings.crop` will work as before.
